clients/fides-js/src/components/Overlay.tsx

```diff
@@ -110,19 +121,20 @@ const Overlay: FunctionComponent<Props> = ({
           options.debug,
           "Modal link element found, updating it to show and trigger modal on click."
         );
+        setIsModalLinkFound(true);
         // Update modal link to trigger modal on click
         const modalLink = modalLinkEl;
-        modalLink.onclick = () => {
-          setBannerIsOpen(false);
-          handleOpenModal();
-        };
+        modalLink.onclick = window.Fides.showModal;
         // Update to show the pre-existing modal link in the DOM
         modalLink.classList.add("fides-modal-link-shown");
       } else {
         debugLog(options.debug, "Modal link element not found.");
       }
     }, delayModalLinkMilliseconds);
-    return () => clearTimeout(delayModalLinkBinding);
+    return () => {
+      clearTimeout(delayModalLinkBinding);
+      window.Fides.showModal = defaultShowModal;
```

Comment:
Sets it back to outputting a debug log about the modal not being supported on cleanup